scripts/apps/archive/services/ArchiveService.ts

```diff
@@ -81,8 +81,7 @@ export function ArchiveService(desks, session, api, $q, search, $location) {
      *  @return {Object} the list of archive items
      */
     this.getRelatedItems = function(item, fromDateTime) {
-        var beforeDateTime = fromDateTime || moment().subtract(1, 'days')
-            .format(appConfig.view.dateformat);
+        var beforeDateTime = formatDate(fromDateTime || moment().subtract(1, 'days'));
```

Comment:
as you can see this function returns `api.query`. Do not change this usage since it is being used for a query, not outputting to user interface.